tests/commonlib/io_utils.py

```diff
@@ -11,6 +12,55 @@
 from munch import Munch, munchify
 
 
+def get_events_from_index(elastic_client, index_name: str, rule_tag: str, time_after: datetime) -> list[Munch]:
```

Comment:
This introduces an additional function that interfaces with ES, I suggest considering moving it to [commonlib/elastic_wrapper.py](https://github.com/elastic/cloudbeat/blob/main/tests/commonlib/elastic_wrapper.py).

In addition, it might be a good idea to patch up the tests in [tests/integration] (https://github.com/elastic/cloudbeat/blob/main/tests/integration/tests/test_cloudbeat_integration.py) to access ES using the same method.

The current convention is a bit confusing.